conjurapi/client.go

```diff
@@ -114,6 +114,61 @@ func NewClientFromEnvironment(config Config) (*Client, error) {
 		return NewClientFromTokenFile(config, authnTokenFile)
 	}
 
+	authnToken := os.Getenv("CONJUR_AUTHN_TOKEN")
+	if authnTokenFile != "" {
+		return NewClientFromToken(config, authnToken)
+	}
+
+	authnJwtServiceID := os.Getenv("CONJUR_AUTHN_JWT_SERVICE_ID")
+	if authnJwtServiceID != "" {
+
+		jwtTokenPath := os.Getenv("JWT_TOKEN_PATH")
+		if jwtTokenPath == "" {
+			jwtTokenPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
Can this string moved to const in another place?

Reply:
Sure.   I can't find any files for variables or constant declaration.   Any suggestions?